packages/core-data/src/utils/get-nested-value.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Helper util to return a value from a certain path of the object.
+ * Path is specified as either:
+ * - a string of properties, separated by dots, for example: "x.y".
+ * - an array of properties, for example `[ 'x', 'y' ]`.
+ * You can also specify a default value in case the result is nullish.
+ *
+ * @param {Object}       object       Input object.
+ * @param {string|Array} path         Path to the object property.
+ * @param {*}            defaultValue Default value if the value at the specified path is nullish.
+ * @return {*} Value of the object property at the specified path.
+ */
+export default function getNestedValue( object, path, defaultValue ) {
```

Comment:
What a bummer that we're introducing yet another version of that utility function. While migrating away from Lodash we had to introduce (almost duplicate) a few of those in various packages, because this was the very last function to remove and we were willing to make some temporary compromises. But ideally, we should unify them all and avoid to introduce new ones if possible.

Reply:
@tyxla I agree. I think we can merge this and follow up with the creation of the package after.Right now we need this util to launch the [Font Library](https://github.com/WordPress/gutenberg/issues/54169). This change is in use here: https://github.com/WordPress/gutenberg/pull/53884